docs/api/modules/App-Configuration.md

```diff
@@ -34,15 +34,16 @@ Node Bundle (e.g.`mymodule.node.js`) rather than the Browser Bundles (e.g.
 security and bundle size considerations.
 
 **Contents**
-* [provideStateConfig](#providestateconfig)
-* [csp](#csp)
-* [corsOrigins](#corsorigins)
-* [configureRequestLog](#configurerequestlog)
-* [extendSafeRequestRestrictedAttributes](#extendsaferequestrestrictedattributes)
-* [createSsrFetch](#createssrfetch)
-* [validateStateConfig](#validatestateconfig)
-* [requiredSafeRequestRestrictedAttributes](#requiredsaferequestrestrictedattributes)
-* [appCompatibility](#appcompatibility)
+- [App Configuration](#app-configuration)
```

Comment:
Why do we need this?

Reply:
I don't remember adding this... Let me remove it